CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+## 0.4.1
+
+- Require the minimum Kotlin version of 1.6.0
+
+## 0.4.0
+
+- Require the minimum Kotlin version of 1.5.30
+- Add support for other Apple Kotlin/Native targets
+- Improve Kotlin/Native support [#24](https://github.com/Kotlin/kotlinx-benchmark/issues/24)
```

Comment:
We can also list the new advanced native option names because it is not clear from the issue.